msresist/figures/figure1.py

```diff
@@ -30,7 +30,7 @@
 def makeFigure():
     """Get a list of the axis objects and create a figure"""
     # Get list of axis objects
```

Comment:
The PCA results are just shoved in at the end, when I think they are really useful to synthesize all the phenotypic measurements. I'd suggest: (1) it's just the PCA results in Fig. 1, with select examples after, and all the phenotypic measurements are in the supplement, or (2) there's a figure on measuring various phenotypes, and then the PCA analysis is a separate Fig. 2.

There probably needs to be a cartoon at the beginning introducing the AXL KO and reintroduction, and small schematics describing each of the phenotypic measurements.